codegens/libcurl/testFile.c

```diff
@@ -0,0 +1,26 @@
+#include <stdio.h>
```

Comment:
Do we need to add this file ? @umeshp7

Reply:
Nope, will delete and include this in gitignore.